taichi/runtime/dx12/aot_module_builder_impl.cpp

```diff
@@ -32,18 +41,91 @@ void AotModuleBuilderImpl::add_field_per_backend(const std::string &identifier,
                                                  std::vector<int> shape,
                                                  int row_num,
                                                  int column_num) {
-  TI_NOT_IMPLEMENTED;
+  // Note that currently we only support adding dense fields in AOT for all
```

Comment:
This comment was probably copied from GFX AOT, where the so-called "all backends" actually refer to "all graphical backends". You can just say "D12 supports only dense fields for now"

Reply:
Btw since D12 comes from LLVM code generation, I think all the sparse fields are easy to support.

Reply:
Updated comments.